assets/js/index.js

```diff
@@ -4,6 +4,12 @@ require('hof-theme-govuk');
 
 document.getElementById('add-image-button').classList.remove('visuallyhidden');
 
+document.getElementById('file-upload').tabIndex = -1;
+
+document.getElementById('image-submit').tabIndex = -1;
+
```

Comment:
Is there a reason why you aren't using `setAttribute`?

Reply:
what is the reason to do it that way? Is it better for readability?